### PR TITLE
Run ExplicitImport.jl's checks on CI

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

File: src/Export/VTK.jl [hunk 1]
```diff
@@ -49,7 +49,7 @@ end
 Base.close(vtk::VTKFile) = WriteVTK.vtk_save(vtk.vtk)
 
 function Base.show(io::IO, ::MIME"text/plain", vtk::VTKFile)
-    open_str = WriteVTK.isopen(vtk.vtk) ? "open" : "closed"
+    open_str = isopen(vtk.vtk) ? "open" : "closed"
```
Comment (KnutAM) [inline src/Export/VTK.jl:52]: Are these mistakes now detected automatically?

File: .github/workflows/Check.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  push:
+    branches: ["master"]
+
+jobs:
+
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pre-commit/action@2c7b3805fd2a0fd8c1884dcaebf91fc102a13ecd # v3.0.1
```
Comment (KnutAM) [inline .github/workflows/Check.yml:14]: Why is there a specific commit here and not `@v3.0.1` ?

**Review by KnutAM — APPROVED** [review]
Changes in code looks good to me, just some curiosity questions.

File: src/Export/VTK.jl (hunk 1 above)
Comment (fredrikekre) [reply src/Export/VTK.jl:52]: Yes :)

File: .github/workflows/Check.yml (hunk 2 above)
Comment (fredrikekre) [reply .github/workflows/Check.yml:14]: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

**Pull request merged**